src/main/java/net/liquidpineapple/pang/screens/Level.java

```
@@ -16,24 +25,111 @@
     private static final int BOTTOM_OFFSET = 10;
 
     public Level() {
+    }
+
+    /**
+     * Method that parses a XML-file into a level.
+     * @param xmlFile - path/filename of the XML-file that should be parsed.
+     * @return - returns a new level.
+     */
+    public static Level createFromXML(String xmlFile) {
 
-        //currentLevel = Level.createFromXML("level1.xml");
+        Level output = new Level();
+        if (createFileReader(xmlFile) != null) {
+            Document doc = createFileReader(xmlFile);
+            try {
+                String background_path = "/sprites/" + doc.getElementsByTagName("background").item(0).getTextContent();
+                output.backgroundImage = ImageIO.read(Level.class.getResource(background_path));
 
-        Player player = new Player(1, 1, 600);
-        objectList.add(player);
-        player.setPos(
-            51, 20
-        );
+                for (Ball ball : loadBalls(doc.getElementsByTagName("ball"))) {
+                    output.objectList.add(ball);
+                }
+
+                for (Player player : loadPlayer(doc.getElementsByTagName("player"))) {
+                    output.objectList.add(player);
+                }
+            }
+            catch (Exception e) {
```

Do we really want to catch ALL exceptions here?